SU2_CFD/src/numerics_structure.cpp

```diff
@@ -332,6 +333,35 @@ void CNumerics::GetInviscidProjJac(su2double *val_velocity, su2double *val_energ
   for (iDim = 0; iDim < nDim; iDim++)
     val_Proj_Jac_Tensor[nDim+1][iDim+1] = val_scale*(val_normal[iDim]*a1-a2*val_velocity[iDim]*proj_vel);
   val_Proj_Jac_Tensor[nDim+1][nDim+1] = val_scale*Gamma*proj_vel;
+
```

Comment:
This change affects the generality of the source code. Do you know how much is worth it in computational time? Thanks!